### PR TITLE
Mining notifications insert fix. Tenantized translations.

### PR DESCRIPTION
**Pull request by ice-myles**
Added time to mining notifications not to have duplicates for sent_notification inserts. Added tenantized translations for scheduler notifications. Urlencode for socials.

### CODE REVIEW

File: notifications/scheduler_announcements.go [hunk 1]
```diff
@@ -96,6 +99,7 @@ func (s *Scheduler) runAnnouncementsProcessor(ctx context.Context, workerNumber
 		******************************************************************************************************************************************************/
 		reqCtx, reqCancel = context.WithTimeout(ctx, requestDeadline)
 		if rErr := runConcurrentlyBatch(reqCtx, s.broadcastPushNotification, toSendAnnouncements, func(arg *broadcastPushNotification[push.Notification[push.SubscriptionTopic]], err error) { //nolint:lll // .
+			log.Error(err, "can't send announcement for lang:%v, notificationChannel:%v, notificationChannelValue:%v, notificationType:%v", arg.sa.Language, arg.sa.NotificationChannel, arg.sa.NotificationChannelValue, arg.sa.NotificationType, arg.sa.NotificationType) //nolint:lll // .
```
Comment (ice-ares) [inline notifications/scheduler_announcements.go]: most likely its rErr, not err

Comment (ice-myles) [reply notifications/scheduler_announcements.go]: No, it' failure callback. And err is arg.

Comment (ice-ares) [reply notifications/scheduler_announcements.go]: also, check all the log.Error calls u added, cuz thats not how that api works. 

It should be
log.Error(errors.Wrap(XXX, "template", args...))

Comment (ice-ares) [reply notifications/scheduler_announcements.go]: ok, then just make sure the api is user properly

**Pull request merged**